packages/framework/src/Migrations/Version20240111072911.php

```diff
@@ -20,14 +19,23 @@ class Version20240111072911 extends AbstractMigration implements ContainerAwareI
      */
     public function up(Schema $schema): void
     {
-        $firstDomainTimeZone = $this->getDomain()->getDomainConfigById(Domain::FIRST_DOMAIN_ID)->getDateTimeZone();
-        $utcTimeZone = new DateTimeZone('UTC');
-        $dateTime = new DateTime('now', $utcTimeZone);
-        $timeOffsetInHours = timezone_offset_get($firstDomainTimeZone, $dateTime) / 3600;
-
-        $this->sql(
-            sprintf('UPDATE blog_articles SET publish_date = publish_date - INTERVAL \'%d hour\';', $timeOffsetInHours),
-        );
+        $blogArticles = $this->sql('SELECT id, publish_date FROM blog_articles')->fetchAllAssociative();
+
+        foreach ($blogArticles as $blogArticle) {
+            $dateTime = new DateTime(
+                $blogArticle['publish_date'],
+                new DateTimeZone('Europe/Prague'),
+            );
+            $dateTime->setTimezone(new DateTimeZone('UTC'));
+
+            $this->sql(
+                'UPDATE blog_articles SET publish_date = :publishDate WHERE id = :id',
+                [
+                    'publishDate' => $dateTime->format('Y-m-d G:i:s'),
+                    'id' => $blogArticle['id'],
+                ],
+            );
```

Comment:
isn't this an appropriate format?

```
$this->sql(
            'UPDATE blog_articles SET publish_date = :publishDate WHERE id = :id',
            [
                'publishDate' => $dateTime,
                'id' => 1,
            ],
            [
                'publishDate' => 'datetime',
                'id' => 'integer',
            ]
        );
        ```

Reply:
of course, nice catch :ok_hand: e2f73a712d